workspaces/src/network/sandbox_shared.rs

```diff
@@ -0,0 +1,125 @@
+use super::{
+    Account, AllowDevAccountCreation, AllowStatePatching, CallExecution, Contract, NetworkClient,
+    NetworkInfo, TopLevelAccountCreator,
+};
+use crate::network::sandbox::NEAR_BASE;
+use crate::network::server::SandboxServer;
+use crate::network::Info;
+use crate::rpc::client::Client;
+use crate::types::{AccountId, Balance, InMemorySigner, SecretKey};
+use async_mutex::Mutex;
+use async_trait::async_trait;
+use once_cell::sync::Lazy;
+use std::path::PathBuf;
+use std::str::FromStr;
+
+static SHARED_SERVER: Lazy<SandboxServer> = Lazy::new(|| {
+    let mut server = SandboxServer::default();
+    server.start().unwrap();
+    server
+});
```

Comment:
Probably used for experimentation but just wanted to note we most likely don't want to start the server before a test starts since that could be problematic if we're running `testnet` while there's a sandbox node spinning up in the background 😰